__tests__/app.test.js

```diff
@@ -174,4 +182,43 @@ describe('POST:201: /api/articles/:article_id/comments', () => {
             expect(body.msg).toBe('Bad request')
         ])
     });
+});
+describe('PATCH: /apr/articles/:article_id', () => {
+    test('PATCH: takes an object with inc_votes, responds with specific article and its votes updated to the given value', () => {
+        return request(app)
+        .patch('/api/articles/1')
+        .send({inc_votes: 10})
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article[0].votes).toBe(110)
+        })
+    });
+    test('PATCH: if passed an object with a - value, responds with specific article and its votes updated to a smaller value', () => {
+        return request(app)
+        .patch('/api/articles/1')
+        .send({inc_votes: -10})
+        .expect(200)
+        .then(({body}) => {
+            expect(body.article[0].votes).toBe(90)
+        })
+    });
+    test('PATCH: if passed an object without a value, responds with an error', () => {
+        return request(app)
+        .patch('/api/articles/1')
+        .send({})

```

Comment:
I would recommend also testing for when passed an object without a valid value on the inc_votes key